protocols/384_plate_filling/384_plate_filling.ot1.py

```diff
@@ -6,18 +6,53 @@
 plate = containers.load('384-plate', 'C1', 'plate')
 
 # 8-channel 10uL pipette, with tip rack and trash
-tiprack = containers.load('tiprack-200ul', 'A1', 'p200rack')
 trash = containers.load('trash-box', 'B2', 'trash')
 
 
-def run_custom_protocol(well_volume: float=1.0, pipette_type: StringSelection(
-    'p300-Single', 'p50-Single', 'p10-Single',
-        'p300-Multi', 'p50-Multi', 'p10-Multi')='p50-Multi'):
+def run_custom_protocol(
+        well_volume: float = 50.0,
+        reagent_well: StringSelection(
+            'A1', 'A2', 'A3', 'A4', 'A5', 'A6',
+            'A7', 'A8', 'A9', 'A10', 'A11', 'A12') = 'A1',
+        plate_starting_column: str = '1',
+        number_of_columns_to_fill: int = 24,
```

Comment:
I think `number_of_columns_to_fill` should be used when defining `alternating_wells` but is missing